src/_data/sidenav.yml

```diff
@@ -502,6 +502,13 @@
       permalink: /packages-and-plugins/using-packages
     - title: Develop packages & plugins
       permalink: /packages-and-plugins/developing-packages
+    - title: Swift Package Manager
+      permalink: /packages-and-plugins/swift-package-manager
+      children:
+        - title: For app developers
+          permalink: /packages-and-plugins/swift-package-manager/for-app-developers
```

Comment:
I'm not sure app developers would think to go into `Packages & plugins`
Maybe this should move into `Platform integration`? Though it would make sense under iOS and macOS, and we don't have a shared category.  Whatever you and the tech writers think works best.

Reply:
oh, good idea!

Reply:
Sorry for the slow response, I'd like to test if I can link to this page twice: once from the packages & plugin section and once from the platform integration section. Will loop back soon!